src/test/java/plugins/MatrixPluginTest.java

```diff
@@ -2,24 +2,63 @@
 
 import com.google.inject.Inject;
 import hudson.util.VersionNumber;
+import org.jenkinsci.test.acceptance.Matcher;
 import org.jenkinsci.test.acceptance.junit.AbstractJUnitTest;
+import org.jenkinsci.test.acceptance.plugins.matrix_reloaded.MatrixReloadedAction;
 import org.jvnet.hudson.test.Issue;
 import org.jenkinsci.test.acceptance.junit.Since;
 import org.jenkinsci.test.acceptance.junit.WithPlugins;
 import org.jenkinsci.test.acceptance.po.*;
 import org.jenkinsci.test.acceptance.slave.SlaveProvider;
 import org.junit.Before;
 import org.junit.Test;
-
 import java.util.List;
 
 import static java.util.Collections.singletonMap;
 import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.CoreMatchers.not;
 import static org.hamcrest.Matchers.is;
 import static org.hamcrest.Matchers.lessThan;
 
-@WithPlugins("matrix-project")
+@WithPlugins({"matrix-project", "matrix-reloaded", "matrix-groovy-execution-strategy"})
```

Comment:
`matrix-reloaded`, `matrix-groovy-execution-strategy` are only needed for some specific tests - can you annotate the test method instead?

Reply:
updated test annotation in b956976d804a295211500307ab166b9619328e0b